packages/clarity-js/src/layout/mutation.ts

```diff
@@ -218,27 +218,27 @@ async function processNodeList(list: NodeList, source: Source, timer: Timer): Pr
   }
 }
 
-function schedule(node: Node): Node {
+export function schedule(node: Node, fragment: boolean = false): Node {
   // Only schedule manual trigger for this node if it's not already in the queue
   if (queue.indexOf(node) < 0) { queue.push(node); }
 
   // Cancel any previous trigger before scheduling a new one.
   // It's common for a webpage to call multiple synchronous "insertRule" / "deleteRule" calls.
   // And in those cases we do not wish to monitor changes multiple times for the same node.
   if (timeout) { clearTimeout(timeout); }
```

Comment:
>if (timeout) { clearTimeout(timeout); } [](http://example.com/codeflow?start=2&length=39)

If we get two mutations which are part of different fragments within LookAhead period; wouldn't we drop capturing the first fragment mutations?